content/en/observability_pipelines/performance.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Performance
+disable_toc: false
+---
+
+## Overview
+
+Sometimes problems can occur even when you try to ensure your Observability Pipelines deployments are sized for the expected load. For example, an application might start generating more logs than usual, or the downstream service you are sending data to starts responding slower than expected. To address these issues:
+- Observability Pipelines propagates backpressure, which is a signal that events cannot be processed as soon as they are received.
+- Observability Pipelines components also have in-memory buffering in case the component receiving the data is busy.
+
+## Backpressure
+
+Backpressure is a signal that events cannot be processed as soon as they are received, and that signal is propagated up through the pipeline. For example, if a processor is sending events to a sink, and the sink cannot process it as fast as it is receiving the events, backpressure is propagated from the sink to the processor and all the way back to the source.
+
+Backpressure determines if the system should slow down the consumption or acceptance of events because it is too busy to handle more work. In some cases though, the system should not immediately propagate backpressure, because this could lead to the constant slowing down of upstream components, and potentially cause issues outside of Observability Pipelines. For example, the system avoids slowing processes down when a component just crosses over the threshold of being fully saturated. The system is also able to handle temporary slowdowns and outages with external services that destinations send data to.
```

Comment:
```suggestion
Backpressure determines if the system should slow down the consumption or acceptance of events because it is too busy to handle more work. In some cases though, the system should not immediately propagate backpressure, because this could lead to the constant slowing down of upstream components, and potentially cause issues outside of Observability Pipelines. For example, the system prevents processes from slowing down when a component barely exceeds the saturation threshold. It can also handle temporary slowdowns and outages with external services that receive data from destinations.
```
Just suggest to re-word this a bit to simplify and make it more clear. (and remove the word "just" that vale is complaining about) :)